shared/teams/confirm-modals/delete-channel/index.tsx

```diff
@@ -69,6 +64,12 @@ const DeleteChannel = (props: Props) => {
         teamID: teamID,
       })
     )
+    dispatch(
```

Comment:
Something you could do, not necessarily that you should do in this PR, is have an action `deleteChannelsConfirmed` where the action handler itself gets the convIDs from the state, so you don't need to pass these. This is fine though.